data-sources/gb-postcodes/README.md

```diff
@@ -0,0 +1,49 @@
+# GB Postcodes
+
+
+The server [importing instructions](https://www.nominatim.org/release-docs/latest/admin/Import-and-Update/) allow optionally download [`gb_postcode_data.sql.gz`](https://www.nominatim.org/data/gb_postcode_data.sql.gz). This document explains how the file got created.
+
+## GB vs UK
+
+GB (Great Britain) is more correct as the Ordnance Survey dataset doesn't contain postcodes from Northern Ireland.
+
+## Importing separately after the initial import
+
+If you forgot to download the file, or have a new version, you can import it separately:
+
+1. Table schema is in `/data/gb_postcode_table.sql`. 
+
+2. Run `GRANT SELECT ON TABLE gb_postcode TO "{www-user}";` (found in file `sql/tables.sql`). `{www-user}` needs to be replaced with the user the webserver (e.g. Apache2) runs under, e.g. on Ubuntu that's `www-data`.
```

Comment:
This shouldn't be needed. Nominatim always creates an empty gb_postcode table and runs the grant on it. So if you import gb_postcpde_data.sql.gz later, that shouldn't affect the permissions.